internal/config/environment.go

```diff
@@ -107,6 +109,8 @@ func ciPlatform() string {
 		return "gitlab_ci"
 	} else if IsTruthy(os.Getenv("CIRCLECI")) {
 		return "circleci"
+	} else if IsTruthy(os.Getenv("SYSTEM_COLLECTIONURI")) {
+		return fmt.Sprintf("azure_devops_%s", os.Getenv("BUILD_REPOSITORY_PROVIDER"))
 	} else {
```

Comment:
Since you touched and fixed few Jenkins-related things. Here's an environment to detect Jenkins: `JENKINS_HOME` has false positives but I think we can ignore them.

Reply:
Done